httpserver/bin/mini_file_server.dart

```diff
@@ -24,7 +24,7 @@ Future main() async {
     print("Couldn't bind to port 4044: $e");
     exit(-1);
   }
-  print('Listening on http://${server.address.address}:${server.port}/');
```

Comment:
```
Error: Could not resolve the package 'path' in 'package:path/path.dart'.
bin/mini_file_server.dart:13:8: Error: Not found: 'package:path/path.dart'
import 'package:path/path.dart' as p;
       ^
bin/mini_file_server.dart:16:17: Error: Method not found: 'dirname'.
    File(p.join(p.dirname(Platform.script.toFilePath()), 'index.html'));
                ^^^^^^^
bin/mini_file_server.dart:16:10: Error: Method not found: 'join'.
    File(p.join(p.dirname(Platform.script.toFilePath()), 'index.html'));
         ^^^^
```

Reply:
Oh interesting! Did you do a `pub get`? (This tutorial is very old and predated `pub`, I think, so it doesn't seem to discuss using `pub` to download the packages you need.)

Reply:
Yes, that is correct, `pub get` is missing from the original tutorial website page. (I am unsure of where those files/the repo is to make an update to the wording include a pub get instruction for the reader).

This highlighted print message was actually specifically for the `server.address.address` and `server.port` types, these require specific packages to be added as well.        

```
mini_file_server.dart:27:39: Error: The getter 'address' isn't defined for the class 'Stream<HttpRequest>'.
 - 'Stream' is from 'dart:async'.
 - 'HttpRequest' is from 'dart:_http'.
Try correcting the name to the name of an existing getter, or defining a getter or field named 'address'.
  print('Listening on http://${server.address}:${server.port}/');
                                      ^^^^^^^
mini_file_server.dart:27:57: Error: The getter 'port' isn't defined for the class 'Stream<HttpRequest>'.
 - 'Stream' is from 'dart:async'.
 - 'HttpRequest' is from 'dart:_http'.
Try correcting the name to the name of an existing getter, or defining a getter or field named 'port'.
  print('Listening on http://${server.address}:${server.port}/');
                                                        ^^^^
```
To resolve (since it is a demo), I added a print line with `localhost:4044` instead (using localhost and port number as a string to avoid types) any advice here? 

I found these to be valuable learnings that could be handled differently for a new user.

Reply:
Going to close the pull request due to the inactivity of discussion on the topic.

Reply:
Updated file server file with proper loopbackIPv4 response